e2e/cypress/integration/accessibility/accessibility_input_fields_spec.js

```diff
@@ -111,6 +111,17 @@ describe('Verify Accessibility Support in different input fields', () => {
     });
 
     it('MM-22625 Verify Accessibility Support in Search Autocomplete', () => {
+        // # Adding at least five other users in the channel
+        cy.getCurrentChannelId().then((channelId) => {
+            cy.getCurrentTeamId().then((teamId) => {
+                cy.createNewUser({}, [teamId]).then((user) => {
+                    cy.apiAddUserToChannel(channelId, user.id);
+                });
+            });
+        });
+        cy.apiLogin('sysadmin');
+        cy.visit('/ad-1/channels/town-square');
```

Comment:
Got these failing on this spec.
![Screen Shot 2020-03-12 at 10 00 19 AM](https://user-images.githubusercontent.com/5334504/76480833-d1e49f00-644a-11ea-9ce2-dc26008519fd.png)

Reply:
The second error isn't happening for me but I updated the way elements are retrieved to see if it passes for you this time

Reply:
The first error is related to loop above

Reply:
I also increased timeout for @ and ~, and made sure navigation arrows are what they are intended to do

![Screen Shot 2020-03-12 at 10 51 13 AM](https://user-images.githubusercontent.com/487991/76550572-94cedb80-644f-11ea-8bf2-395f4966bfe4.png)

Reply:
Thanks, all passed now.